programs/programs/tx/fpp/tests/test_fpp.py

```
@@ -142,34 +165,124 @@ def test_employer_insurance_does_not_disqualify(self):
+        self.assertEqual(calc._countable_income(), 0)
+
+    def test_member_with_unknown_age_earnings_excluded(self):
+        """A member with no recorded age is not treated as an adult, so their earnings are
+        excluded (can't confirm they're 18+)."""
+        adult = make_member(age=40, earned=20_000)
+        unknown_age = make_member(age=None, earned=15_000)
+        calc = make_calculator(fpl_limit=15_000, members=[adult, unknown_age])
+        self.assertEqual(calc._countable_income(), 20_000)
+
+    def test_child_support_paid_and_received_combined(self):
+        """Both child-support branches apply together: $20k unearned + ($10k received −
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)